packages/js/src/elementor/replaceVars/elementor-replacevar-plugin.js

```diff
@@ -58,6 +63,7 @@ const getReplacements = ( scope = "" ) => {
 				"primaryCategory",
 				"pageNumber",
 				"pageTotal",
+				"postContent",
```

Comment:
should this be with our without an underscore?

Reply:
The way the replace vars work in Elementor is a bit different. 

These names correspond with the filenames of the replace vars (located in `js/src/elementor/replaceVars/general`) instead of the names of the replace vars themselves (with the underscores). 

Since the other replace var filenames are in camelCase, I opted to use camelCase for the new replace vars as well.